### PR TITLE
Add Smack (XMPP client library)

### PR DESCRIPTION
**Pull request by Flowdalic**

### CODE REVIEW

File: _data/projects/Smack.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  Smack is an open source, highly modular, easy to use, XMPP client library written in Java for Java SE compatible JVMs and Android.
+site: https://www.igniterealtime.org/projects/smack/
+tags:
+- xmpp
+- java
+- im
+- client
+upforgrabs:
+  name: Teaser Tasks
+  link: https://igniterealtime.atlassian.net/issues/?filter=10003
```
Comment (shiftkey) [inline _data/projects/Smack.yml:13]: Not a blocker on merging, but when I follow this link I get this error:

`The requested filter doesn't exist or is private. Try logging in.`

Does that sound right?

Comment (Flowdalic) [reply _data/projects/Smack.yml:13]: The JIRA filter should be public now.

Comment (shiftkey) [reply _data/projects/Smack.yml:13]: Still no change on my end 🤔

Comment (Flowdalic) [reply _data/projects/Smack.yml:13]: Now it should work. Sorry.

**Review by shiftkey — APPROVED** [review]
Thanks for the contribution @Flowdalic!

**Pull request merged**